mungegithub/github/github.go

```diff
@@ -57,6 +56,10 @@ const (
 	headerRateReset     = "X-RateLimit-Reset"
 
 	maxCommentLen = 65535
+
+	// Option keys.
+	keyOrg     = "organization"
```

Comment:
probably not worth splitting this out for a string used once in code and once in a help message